templates/acs-deployment-master.yaml

```diff
@@ -371,9 +371,9 @@ Parameters:
       NoEcho: True
     RegistryCredentials:
       Type: String
-      Description: The private registry credentials base64
+      Description: The private registry credentials base64.  Default value provided is an example in base64 encoded.
       NoEcho: True
-      Default: ""
+      Default: "ewoJImF1dGhzIjogewoJCSJodHRwczovL3ByaXZhdGVyZWdpc3RyeS5jb20iOiB7CgkJCSJhdXRoIjogImRHVnpkRUIwWlhOMExtTnZiVHB3WVhOemQyOXlaQW89IgoJCX0sCgl9LAoJIkh0dHBIZWFkZXJzIjogewoJCSJVc2VyLUFnZW50IjogIkRvY2tlci1DbGllbnQvMTguMDYuMS1jZSAoZGFyd2luKSIKCX0sCgkic3RhY2tPcmNoZXN0cmF0b3IiOiAia3ViZXJuZXRlcyIKfSUK"
```

Comment:
This was set to an empty string to avoid committing something that looks like real credentials, see the comments on https://github.com/Alfresco/acs-deployment-aws/pull/11 for the discussion.

If this results in an error (it didn't in my testing) we should add Conditions to exclude the registry details when set to "".

Reply:
I tried with literal string (ex: `false`) or leaving it as empty `""` but it throws below exception:
```
Error from server (BadRequest): error when creating "secret.yaml": Secret in version "v1" cannot be handled as a Secret: v1.Secret: ObjectMeta: v1.ObjectMeta: TypeMeta: Kind: Data: decode base64: illegal base64 data at input byte 0, error found in #10 byte of ...|--install"},"kind":"|..., bigger context ...|sion":"v1","data":{".dockerconfigjson":"--install"},"kind":"Secret","metadata":{"name":"quay-registr|...
```

Reply:
I will try to condition the `RegistryCredential` so if it is set to empty value `""`, don't create the secret.yaml, else create the secret.

Reply:
Yes, that would be ideal!